src/components/nav-drop-down.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
```

Comment:
Good idea to create a new component for the nav dropdown! I was just thinking about that too